src/lcm-drive.py

```diff
@@ -0,0 +1,85 @@
+from robot import dmw_position_t, dmw_goal_position_t
+from __future__ import print_function   # use python 3 syntax but make it compatible with python 2
+from __future__ import division         # 
```

Comment:
We do not need to support Python 2. It reached end-of-life earlier this year.

Also, add a `#!/usr/bin/env python3` shebang.

Reply:
answer to your question: lcm-drive.py the DWM mode node